inventories/kronos/group_vars/cloud_snitch_sync.yml

```diff
@@ -0,0 +1,7 @@
+cloud_snitch_neo4j_enabled: True
+cloud_snitch_neo4j_password: "{{ pwsafe_neo4j_password }}"
+cloud_snitch_neo4j_username: "{{ pwsafe_neo4j_username }}"
+cloud_snitch_neo4j_uri: bolt://172.16.10.5
```

Comment:
We probably should keep actual locations out of inventory also. Maybe this should be an address from host vars of the neo4j instance.

Reply:
Good feedback.  Latest push fixes this.